boxoffice/views/admin_report.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from flask import Response, jsonify
+from .. import app, lastuser
+from coaster.views import load_models, render_with
+from boxoffice.models import db, ItemCollection, Order, Item, LineItem, Assignee, DiscountPolicy, LINE_ITEM_STATUS
+
+
+def jsonify_report(data_dict):
+    return jsonify(org_name=data_dict['item_collection'].organization.name, title=data_dict['item_collection'].title)
+
+
+@app.route('/admin/ic/<ic_id>/reports')
+@lastuser.requires_login
+@load_models(
+    (ItemCollection, {'id': 'ic_id'}, 'item_collection'),
+    permission='org_admin'
+    )
+@render_with({'text/html': 'index.html', 'application/json': jsonify_report}, json=True)
```

Comment:
Don't use `json=True` if you're providing an explicit `application/json` handler. It provides another for `text/json` which can be a backdoor into your code.